package-lock.json

```diff
@@ -15170,7 +15170,7 @@
     "array-flatten": {
       "version": "1.1.1",
       "resolved": "https://registry.npmjs.org/array-flatten/-/array-flatten-1.1.1.tgz",
-      "integrity": "sha1-ml9pkFGx5wczKPKgCJaLZOopVdI=",
+      "integrity": "sha512-PCVAQswWemu6UdxsDFFX/+gVeYqKAod3D3UVm91jHwynguOwAvYPhx8nNlM++NqRcK6CxxpUafjmhIdKiHibqg==",
```

Comment:
Could these changes to `package-lock.json` cause any issues? I'm approving the PR, but deferring this to you, as it seems a bit strange to change a few integrity hashes to sha512 while leaving some as sha1.

Reply:
I wouldn't expect issues from this as there are several references of sha512 and sha1 in this file.